cni/pkg/install-cni/cmd/root.go

```diff
@@ -138,6 +138,7 @@ func constructConfig() (*config.Config, error) {
 
 		UpdateCNIBinaries: viper.GetBool(constants.UpdateCNIBinaries),
 		SkipCNIBinaries:   viper.GetStringSlice(constants.SkipCNIBinaries),
+		CNIBinariesPrefix: viper.GetString(constants.CNIBinariesPrefix),
```

Comment:
I think you need to register it in `init()`

Reply:
Thanks! Fixed.